activerecord/lib/active_record/base.rb

```diff
@@ -1652,13 +1652,13 @@ def find_one(id, options)
           if result = find_every(options).first
             result
           else
-            raise RecordNotFound, "Couldn't find #{name} with ID=#{id}#{conditions}"
+            raise RecordNotFound, "Couldn't find #{name} with ID=#{id.inspect}#{conditions}"
           end
         end
 
         def find_some(ids, options)
           conditions = " AND (#{sanitize_sql(options[:conditions])})" if options[:conditions]
-          ids_list   = ids.map { |id| quote_value(id,columns_hash[primary_key]) }.join(',')
+          ids_list   = ids.map { |id| quote_value(id.inspect,columns_hash[primary_key]) }.join(',')
```

Comment:
Seems it's not necessary here because it's using `quote_value`
[#quote_value](https://github.com/Genius/rails2/blob/24a9b7b409d445ccde50ce848dc6c9b53b6ce0d0/activerecord/lib/active_record/base.rb#L1481) > [connection#quote](https://github.com/Genius/rails2/blob/24a9b7b409d445ccde50ce848dc6c9b53b6ce0d0/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L370)